test/helpers/reducers.js

```diff
@@ -30,3 +30,22 @@ export function todosReverse(state = [], action) {
     return state;
   }
 }
+
+export function antipatternTodo(state = [], action) {
```

Comment:
Maybe call it `dispatchInTheMiddleOfReducer`? It isn't much related to todos anyway.